spec/function.dd

```diff
@@ -1234,20 +1234,42 @@ int foo(in int x, out int y, ref int z, int q);
     implemented so it's current implementation is equivalent to $(D const).  It is recommended
     to avoid using $(D in) until it is properly defined and implemented.  Use $(D scope const)
     or $(D const) explicitly instead.)
-    $(TROW $(D out), parameter is initialized upon function entry with the default value
+    $(TROW $(D ref), parameter is passed by reference)
+    $(TROW $(D out), parameter is passed by reference and initialized upon function entry with the default value
     for its type)
 
-    $(TROW $(D ref),   parameter is passed by reference)
-    $(TROW $(D scope), references in the parameter
+    $(TROW $(D scope), references to the parameter
```

Comment:
I presume the original was just a mistake, as `scope` is not transitive anyway.

Reply:
It doesn't seem to make sense either way.  I recommend not even including it in this PR.  Leave it for another PR as it might derail the progress on this PR.

Reply:
@JinShil OK, moved to #2454.